### ISSUE TITLE
Davinci Resolve 19 installation issues

### ISSUE THREAD

**Issue by LucasGGamerM** [zelikos/davincibox#122]
**IMPORTANT**

Before reporting a bug, make sure that you can reproduce it on the latest version of davincibox.

See the [upgrading](https://github.com/zelikos/davincibox#upgrading) section of the README for more information.

**Describe the bug**

**Installation method used**

- [x ] `setup.sh`
- Manual
  - [ ] Distrobox
  - [ ] Toolbox

**To Reproduce**

Launch the setup with the installer passed to it,
Wait until it's done,
See the error in the bottom: 
```
squashfs-root/share/resolve.xml
egrep: warning: egrep is obsolescent; using grep -E
Error: Missing or outdated system packages detected.

Please install the following missing packages:
    xcb-util-cursor

Use SKIP_PACKAGE_CHECK=1 to bypass the system package check.

***********************

```

**Expected behavior**

It installing properly.

**Screenshots**

**System information (please complete the following)**

 - Distro: PopOS 22.04
 - Desktop Environment: Gnome
 - GPU: rtx 3050

**Additional context**

**Comment by zelikos** [comment]
I can't reproduce this error on my end, but I can confirm that davincibox does currently lack the `xcb-util-cursor` dependency. I'll add it to the list

I don't know how this error could have popped up to begin with either, as `SKIP_PACKAGE_CHECK=1` was applied as a [workaround](https://github.com/zelikos/davincibox/pull/67) several months back. Even without that environment variable, though, I don't get the error here.

**Comment by LucasGGamerM** [comment]
If it is relevant, I am using Davinci Resolve Studio, and the version I tried to install was DaVinci_Resolve_Studio_19.0.1_Linux

**Comment by zelikos** [comment]
I don't know if Studio would have different dependencies than the free version, but regardless, I'm adding the package in #123.

**Comment by LucasGGamerM** [comment]
This sadly still happens in my case. I wonder if it has got to do with maybe my messy system. Is there any other info I can provide you to better help you solve this issue?

**Comment by zelikos** [comment]
After upgrading davincibix, you have the exact same issue as originally posted about `xcb-util-cursor` missing?

**Comment by LucasGGamerM** [comment]
> After upgrading davincibox, you have the exact same issue as originally posted about `xcb-util-cursor` missing?

Precisely. Its the exact same behavior. I saw it downloaded a new image, so I am sure davincibox at least tried updating itself.

**Comment by zelikos** [comment]
In that case, I'd be curious about what you meant by "messy system."

The first thing I noticed is that you're using PopOS 22.04. My first suspect would be that the older Ubuntu base means an old version of podman, but I've had success before with testing davincibox on both Linux Mint 21.x and elementary OS 7, both based on the same Ubuntu version. I'll test it on PopOS when I'm able to.

My second suspect: distrobox can use more than just podman for its backend, such as docker. If distrobox is using a different backend that could also be an issue; davincibox has only been tested with podman as the backend.

**Comment by LucasGGamerM** [comment]
By messy system, I mean that I have had it installed for more than a year, and with a ton of extra installed packages. 

When I get home I will assert whether or not I am using podman.

As for testing, could you try testing it with the Studio version?

**Comment by kamikazebr** [comment]
> By messy system, I mean that I have had it installed for more than a year, and with a ton of extra installed packages.
> 
> When I get home I will assert whether or not I am using podman.
> 
> As for testing, could you try testing it with the Studio version?

was solved? I want to buy Studio but and using PopOS 22.04 too and Community versions works normal.

Maybe i should go to another distro, if someone know the best distro to run Studio, let me know.

**Comment by kamikazebr** [comment]
> > > By messy system, I mean that I have had it installed for more than a year, and with a ton of extra installed packages.
> > > When I get home I will assert whether or not I am using podman.
> > > As for testing, could you try testing it with the Studio version?
> > 
> > 
> > was solved? I want to buy Studio but and using PopOS 22.04 too and Community versions works normal.
> > Maybe i should go to another distro, if someone know the best distro to run Studio, let me know.
> 
> If the free version works for you, then I believe the studio version will also work.
> 
> I have since switched back to fedora kde, and davinci resolve 19 works well now through davincibox

It's works, even without davincibox too, but cannot change resolutin without that been cut in the screen. Tried a lot things without sucess.
